boards/x86/up_squared/doc/up_squared.rst

```diff
@@ -300,6 +299,135 @@ Steps
          WARNING: no console will be available to OS
 
 
+Booting the UP Squared Board over network
+=========================================
+
+Build Zephyr image
+------------------
+
+#. Follow `Build Zephyr application`_ steps to build Zephyr image.
+
+Prepare Linux host
+------------------
+
+#. Follow `Creating a GRUB2 Boot Loader Image from a Linux Host`_ steps
+   to create grub binary.
+
+#. Install DHCP, TFTP servers. For example `dnsmasq`
+
+   .. code-block:: console
+
+      $ sudo apt-get install dnsmasq
+
+#. Configure DHCP server. Configuration for `dnsmasq` is below:
+
+   .. code-block:: console
+
+      # Only listen to this interface
+      interface=eno2
+      dhcp-range=10.1.1.20,10.1.1.30,12h
+
+#. Configure TFTP server.
+
+   .. code-block:: console
+
+      # tftp
+      enable-tftp
+      tftp-root=/srv/tftp
+      dhcp-boot=grub_x86_64.efi
+
+   `grub_x86_64.efi` is a grub binary created above.
+
+#. Create the following directories inside TFTP root :file:`/srv/tftp`
+
+   :file:`EFI`
+
+   :file:`EFI/BOOT`
+
+   :file:`kernel`
+
+#. Copy the Zephyr image :file:`zephyr/zephyr.strip` to the
+   :file:`$TFTP/kernel` folder.
+
+#. Copy your built version of GRUB to :file:`$TFTP/grub_x86_64.efi`
+
+#. Create :file:`$TFTP/EFI/BOOT/grub.cfg` containing the following:
+
+   .. code-block:: console
+
+      set default=0
+      set timeout=10
+
+      menuentry "Zephyr Kernel" {
+         multiboot /kernel/zephyr.strip
+      }
+
+#. TFTP root should be looking like:
+
+   .. code-block:: console
+
+      $ tree /srv/tftp
+      /srv/tftp
+      ├── EFI
+      │   └── BOOT
+      │       └── grub.cfg
+      ├── grub_x86_64.efi
+      └── kernel
+          └── zephyr.strip
+
+#. Restart `dnsmasq` service:
+
+   .. code-block:: console
+
+      $ sudo systemctl restart dnsmasq.service
+
+Prepare UP Squared board for network boot
+-----------------------------------------
+
+#. Enable PXE network from BIOS settings.
+
+   .. code-block:: console
+
+      Advanced -> Network Stack Configuration -> Enable Network Stack -> Enable Ipv4 PXE Support
+
+#. Make network boot as the first boot option.
+
+   .. code-block:: console
+
+      Boot -> Boot Option #1 : [Network]
+
+Booting UP Squared
+------------------
+
+#. Connect the board to the host system using the serial cable and
+   configure your host system to watch for serial data.  See
+   https://wiki.up-community.org/Serial_console.
+
+   .. note::
+      On Windows, PuTTY has an option to set up configuration for
```

Comment:
Don't all serial console apps have this configuration?

Reply:
I will remove `note`